src/core/handle.ml

```diff
@@ -187,6 +197,35 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
       let d = if op then None else Some(t) in
       let ss = Sig_state.add_symbol ss e Defin x a impl d in
       (ss, None)
+  | P_inductive(e, s, t, tl)     ->
+      (* Obtaining of the global type *)
+      let type_symbol_head = match t with
+        | None   -> none P_Type
```

Comment:
We should probably make the term t mandatory at the parsing level because there is no reason to set this type to TYPE.

Reply:
It's like in Coq, no?

Reply:
LP is not Coq.